Cannot run gdx tests with lwjgl3 on OS X

#### Reproduction steps/code
Initialize everything needed to [run tests from source](https://github.com/libgdx/libgdx/wiki/Running-demos-%26-tests),  
`gradle tests:gdx-tests-lwjgl:launchTestsLwjgl` should work fine.

Then run `gradle tests:gdx-tests-lwjgl3:launchTestsLwjgl3` on OS X 10.11 (or maybe any OS X), this is going to fail.

> GLFW windows may only be created on the main thread and that thread must be the first thread in the process. Please run the JVM with -XstartOnFirstThread. For offscreen rendering, make sure another window toolkit (e.g. AWT or JavaFX) is initialized before GLFW.

#### Version of LibGDX and/or relevant dependencies
adb26f3b776fca66db14d94e4a67d2ff6d9382fd and everything as referenced

#### Stacktrace
```java
:tests:gdx-tests-lwjgl3:launchTestsLwjgl3
Exception in thread "main" java.lang.ExceptionInInitializerError
        at org.lwjgl.glfw.GLFW.glfwCreateWindow(GLFW.java:1384)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createGlfwWindow(Lwjgl3Application.java:425)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createWindow(Lwjgl3Application.java:372)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:107)
        at com.badlogic.gdx.tests.lwjgl3.Lwjgl3TestStarter.main(Lwjgl3TestStarter.java:45)
Caused by: java.lang.IllegalStateException: GLFW windows may only be created on the main thread and that thread must be the first thread in the process. Please run the JVM with -XstartOnFirstThread. For offscreen rendering, make sure another window toolkit (e.g. AWT or JavaFX) is initialized before GLFW.
        at org.lwjgl.glfw.EventLoop$OffScreen.<clinit>(EventLoop.java:38)
        ... 5 more
AL lib: (EE) alc_cleanup: 1 device not closed
```

#### affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [x] MacOS

--- a/tests/gdx-tests-lwjgl3/build.gradle
+++ b/tests/gdx-tests-lwjgl3/build.gradle
@@ -34,6 +34,11 @@ task launchTestsLwjgl3 (dependsOn: classes, type: JavaExec) {
     standardInput = System.in
     workingDir = new File("../gdx-tests-android/assets")
     ignoreExitValue = true
+    
+    if (System.getProperty("os.name").toLowerCase().contains("mac")) {
+        // Required to run lwjgl java apps on Mac OSX
+        jvmArgs = ["-XstartOnFirstThread"]
+    }
 }
 configure (launchTestsLwjgl3) {
     group "LibGDX"

